holoviews/operation/element.py

```diff
@@ -545,13 +545,15 @@ class contours(Operation):
 
     def _process(self, element, key=None):
         try:
-            from matplotlib.contour import QuadContourSet
-            from matplotlib.axes import Axes
-            from matplotlib.figure import Figure
             from matplotlib.dates import num2date, date2num
+            from matplotlib.ticker import MaxNLocator
         except ImportError:
             raise ImportError("contours operation requires matplotlib.")
-        extent = element.range(0) + element.range(1)[::-1]
```

Comment:
This is no longer used as I don't think it is required. There is explicit code to set the `x` and `y` arrays and pass them through, so the `extent` kwarg would have been ignored.